test/org/javarosa/xform/parse/XFormParserTest.java

```diff
@@ -105,7 +107,11 @@ public void parsesForm2() throws IOException {
 
     @Test
     public void parsesPreloadForm() throws IOException {
-        FormDef formDef = parse(r("Sample-Preloading.xml"));
+        // The form on this test uses a jr://file-csv resource.
+        // We need to prime the ReferenceManager to deal with those
+        Path form = r("Sample-Preloading.xml");
```

Comment:
@dcbriccetti @ggalmazor This particular form doesn't actually use the instance loaded by `file-csv` because it relies on the Collect-only database-backed implementation of the `search()` appearance/function: https://github.com/opendatakit/collect/blob/d52ce0bfc63a11fb7f3fc62e0c0fd7c58684527a/collect_app/src/main/java/org/odk/collect/android/external/handler/ExternalDataHandlerSearch.java. You'll see the form doesn't actually use the instance. It's not a big deal since this test only verifies that the form can be parsed but it would probably be worth changing to avoid confusion.

I would also recommend avoiding the "preload' word unless referring to https://opendatakit.github.io/xforms-spec/#preload-attributes

Reply:
We should file an issue for this change

Reply:
I propose removing that test in https://github.com/opendatakit/javarosa/pull/452